File: TunnelProject/Opc/OpcIncomingData.cs
namespace TunnelProject.Opc
{
    public class OpcIncomingData
    {
        #region Fields
        private int _transactionID;
        private int _groupHandle;
        private int _numItems;
        private Array _itemValues;
        private Array _clientHandles;
        private Array _timeStamps;
        private Array _qualities;
        #endregion

        #region Properties
        public Array Qualities
        {
            get { return _qualities; }
            set { _qualities = value; }
        }

        public int TransactionID
        {
            get { return _transactionID; }
            set { _transactionID = value; }
        }

        public int GroupHandle
        {
            get { return _groupHandle; }
            set { _groupHandle = value; }
        }

        public int NumItems
        {
            get { return _numItems; }
            set { _numItems = value; }
        }

        public Array ItemValues
        {
            get { return _itemValues; }
            set { _itemValues = value; }
        }

        public Array ClientHandles
        {
            get { return _clientHandles; }
            set { _clientHandles = value; }
        }

        public Array TimeStamps
        {
            get { return _timeStamps; }
            set { _timeStamps = value; }
        }
        #endregion

        #region Constructors
        public OpcIncomingData(int transactionID, int groupHandle, Array clientHandles, int numItems, Array itemValues, Array timeStamps, Array qualities)
        {
            _transactionID = transactionID;
            _groupHandle = groupHandle;
            _numItems = numItems;
            _itemValues = itemValues;
            _clientHandles = clientHandles;
            _timeStamps = timeStamps;
            _qualities = qualities;
        }
        #endregion
    }
}

File: TSA_Rail_System_TunnelProject/Wdw_Sust.OpcDa.Core/OpcIncomingData.vb
Class OpcIncomingData

#Region "Fields"

    Private _clientHandles As Array
    Private _groupHandle As Integer
    Private _itemValues As Array
    Private _numItems As Integer
    Private _qualities As Array
    Private _timeStamps As Array
    Private _transactionID As Integer

#End Region

#Region "Properties"

    Public Property ClientHandles As Array
        Get
            Return _clientHandles
        End Get
        Set(value As Array)
            _clientHandles = value
        End Set
    End Property

    Public Property GroupHandle As Integer
        Get
            Return _groupHandle
        End Get
        Set(value As Integer)
            _groupHandle = value
        End Set
    End Property

    Public Property ItemValues As Array
        Get
            Return _itemValues
        End Get
        Set(value As Array)
            _itemValues = value
        End Set
    End Property

    Public Property NumItems As Integer
        Get
            Return _numItems
        End Get
        Set(value As Integer)
            _numItems = value
        End Set
    End Property

    Public Property Qualities() As Array
        Get
            Return _qualities
        End Get
        Set(value As Array)
            _qualities = value
        End Set
    End Property

    Public Property TimeStamps As Array
        Get
            Return _timeStamps
        End Get
        Set(value As Array)
            _timeStamps = value
        End Set
    End Property

    Public Property TransactionID As Integer
        Get
            Return _transactionID
        End Get
        Set(value As Integer)
            _transactionID = TransactionID
        End Set
    End Property

#End Region

#Region "Constructors"

    Public Sub New(transactionID As Integer, groupHandle As Integer, clientHandles As Array, numItems As Integer, itemValues As Array, timeStamps As Array, qualities As Array)
        _transactionID = transactionID
        _groupHandle = groupHandle
        _numItems = numItems
        _itemValues = itemValues
        _clientHandles = clientHandles
        _timeStamps = timeStamps
        _qualities = qualities
    End Sub

#End Region

End Class